src/Coordination/KeeperDispatcher.cpp

```diff
@@ -368,9 +369,15 @@ void KeeperDispatcher::initialize(const Poco::Util::AbstractConfiguration & conf
     snapshot_thread = ThreadFromGlobalPool([this] { snapshotThread(); });
 
     snapshot_s3.startup(config, macros);
-
     keeper_context = std::make_shared<KeeperContext>(standalone_keeper);
-    keeper_context->initialize(config, this);
+    String availability_zone;
+    try{
+        availability_zone = DB::S3::getRunningAvailabilityZone();
+    } catch (...)
```

Comment:
By our style should be like this:
```suggestion
    try
    {
        availability_zone = DB::S3::getRunningAvailabilityZone();
    } 
    catch (...)
```